rules-java/api/src/main/java/org/jboss/windup/rules/apps/mavenize/ArchiveCoordinateService.java

```diff
@@ -1,16 +1,17 @@
 package org.jboss.windup.rules.apps.mavenize;
 
-
 import java.util.Iterator;
 import java.util.logging.Logger;
+
```

Comment:
I wish that we were consistent in formatting them. I don't see it as a big issue either way, though. The auto-formatters seem to behave somewhat inconsistently.